src/components/TreeFilter/VirtualizedTreeView.tsx

```diff
@@ -181,6 +181,7 @@ export class VirtualizedTreeView extends React.PureComponent<IVirtualizedTreeVie
                             rowRenderer={this.rowRenderer}
                             rowCount={this.state.filteredItems.length}
                             scrollToIndex={this.state.scrollToIndex}
+                            {...this.props.filterSelection}

```

Comment:
is the forceUpdate in componentDidUpdate still relevant? or can it be removed now?

Reply:
yeah, i think it can go now